modules/sfp_dnsbrute.py

```diff
@@ -30,212 +30,396 @@ class sfp_dnsbrute(SpiderFootPlugin):
 
     # Default options
     opts = {
-        "skipcommonwildcard": True,
         "domainonly": True,
         "commons": True,
         "top10000": False,
-        "numbersuffix": True,
-        "numbersuffixlimit": True,
+        "numbermutation": True,
+        "alphamutation": True,
         "_maxthreads": 100
     }
 
     # Option descriptions
     optdescs = {
-        'skipcommonwildcard': "If wildcard DNS is detected, don't bother brute-forcing.",
-        'domainonly': "Only attempt to brute-force names on domain names, not hostnames (some hostnames are also sub-domains).",
-        'commons': "Try a list of about 750 common hostnames/sub-domains.",
-        'top10000': "Try a further 10,000 common hostnames/sub-domains. Will make the scan much slower.",
-        'numbersuffix': "For any host found, try appending 1, 01, 001, -1, -01, -001, 2, 02, etc. (up to 10)",
-        'numbersuffixlimit': "Limit using the number suffixes for hosts that have already been resolved? If disabled this will significantly extend the duration of scans.",
+        "domainonly": "Only attempt to brute-force names on domain names, not hostnames (some hostnames are also sub-domains).",
+        "commons": "Try a list of about 750 common hostnames/sub-domains.",
+        "top10000": "Try a further 10,000 common hostnames/sub-domains.",
+        "numbermutation": "For any host found, increment/decrement existing numbers (if any) and try appending 1, 01, 001, -1, -01, -001, 2, 02, etc. (up to 10)",
+        "alphamutation": "For any host found, try common mutations such as -test, -old, etc.",
         "_maxthreads": "Maximum threads"
     }
 
-    events = None
-    sublist = None
-    lock = None
-
     def setup(self, sfc, userOpts=dict()):
+
```

Comment:
Not a fan of the style having an empty newline after the function definition - could you please re-style to fit with the convention of the rest of the codebase?